custom_components/miele/translations/de.json

```diff
@@ -294,14 +294,14 @@
                     "white_tea": "Weißer Tee",
                     "japanese_tea": "Japanischer Tee",
                     "green_tea": "Grüner Tee",
-                    "fruit_tea": "Früchtetee",
-                    "herbal_tea": "Kräutertee",
+                    "fruit_tea": "Früchte-Tee",
+                    "herbal_tea": "Kräuter-Tee",
```

Comment:
As above, I think deviating from Miele's choice (in one coffee machine) is better aesthetically: `Kräutertee`.